### ISSUE TITLE
Frequency problem

### ISSUE THREAD

**Issue by bbevren** [Redfluffydragon/tuner#1]
Hi, ui looking great but i have tried with my ukulele. Frequency and displayed notes are not correct, it is way off.  Can you use some kind of model such as https://marl.github.io/crepe/

**Comment by Redfluffydragon** [comment]
There was a problem with calculating the letter note and octave below A4, but I've fixed that now. I also did a bunch of testing with a frequency generator, and the frequency should be (at least approximately) correct with just a single tone down to around A2, where in my testing it started reading an octave higher and I'm not sure why. Also, on something like a piano where there are additional frequencies octaves apart, it likes to jump octaves, so I'll have to look into that.

I have also now implemented a quadratic/parabolic interpolation algorithm. As far as I can tell this is the best and easiest interpolation algorithm without having access to the imaginary parts of an FFT, which the Web Audio API does not give you.
